src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts

```diff
@@ -7,8 +7,8 @@ import {
   AfterViewInit,
   Component,
   OnInit,
-  ViewChild,
   signal,
+  ViewChild,
```

Comment:
Is this order valid with our lint settings? Please check by running `./gradlew check`

Reply:
gradlew says 👌